learn/guides/ballerina-shell.md

```diff
@@ -78,53 +97,50 @@ Any valid Ballerina statement can be evaluated in the Shell. The example below e
 An expression always produces a value when evaluated. You can use the Shell to evaluate any arbitrary expression. It will output the result of the evaluation.
 
 ```bash
-=$ 10 + 20 * 5                                                                                                                                                                                                       
+=$ 10 + 20 * 5
 110
 ```
 
 A reference to a variable is also an expression. Thus, evaluating the name of the previously-defined variable will return its value.
 
 ```bash
-=$ name                                                                                                                                                                                                            
+=$ name
 "Ballerina Shell"
 ```
 
-###  Defining Types
+### Defining Variables
 
-Type definitions can be written just as you write them in a source file. The Shell supports multi-line snippets.
+Variables defined in the Shell prompt will be treated as module-level variables. Once defined, these can be used in any subsequent evaluation.
 
 ```bash
-=$ type Person record {                                                                                                                                                                                              
- > string name;
- > int age;
- > }
+=$ int a = 10  
 ```
 
-Execute the `/dclns` command to view the defined types (and other module-level declarations).
+Execute the `/vars` command as shown below to view the declared variables.
 
 ```bash
-=$ /dclns                                                                                                                                                                                                          
-| ('Person) type Person record {string name;int age;};
+=$ /vars
+| (a) int a = 10
 ```
 
-### Defining Variables
+### Remove Variables
 
-Variables defined in the Shell prompt will be treated as module-level variables. Once defined, these can be used in any subsequent evaluation.
+`/remove` command can be used to remove declarations.
```

Comment:
```suggestion
The `/remove` command can be used to remove declarations.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75